### PR TITLE
CON-1712 tech dept

### PR DESCRIPTION
**Pull request by shomavg**

### CODE REVIEW

File: src/SFA.DAS.EAS.Employer_Account.Database/Scripts/Manual/CON-1712-alterTable.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE [employer_account].[EmployerAgreementTemplate]
```
Comment (cofaulco) [inline src/SFA.DAS.EAS.Employer_Account.Database/Scripts/Manual/CON-1712-alterTable.sql:1]: No need for this script. simply open the EmployerAgreementTemplate.sql and add the column

File: src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs [hunk 2]
```diff
@@ -14,14 +14,19 @@ public AgreementMappings()
...
-                .ForMember(dest => dest.TemplatePartialViewName, opt => opt.MapFrom(src => src.Template.PartialViewName))
+                .ForMember(dest => dest.TemplatePartialViewName,
+                    opt => opt.MapFrom(src => src.Template.PartialViewName))
                 .ForMember(dest => dest.AgreementType, opt => opt.MapFrom(src => src.Template.AgreementType))
-                .ForMember(dest => dest.AccountLegalEntityId, opt => opt.MapFrom(src => src.LegalEntity.AccountLegalEntityId))
-                .ForMember(dest => dest.AccountLegalEntityPublicHashedId, opt => opt.MapFrom(src => src.LegalEntity.AccountLegalEntityPublicHashedId))
+                .ForMember(dest => dest.AccountLegalEntityId,
+                    opt => opt.MapFrom(src => src.LegalEntity.AccountLegalEntityId))
+                .ForMember(dest => dest.AccountLegalEntityPublicHashedId,
+                    opt => opt.MapFrom(src => src.LegalEntity.AccountLegalEntityPublicHashedId))
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs:29]: can we undo these newline changes as they bare no relevance to the code change

File: src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs [hunk 3]
```diff
@@ -39,28 +44,13 @@ public AgreementMappings()
...
 
             CreateMap<EmployerAgreementDto, OrganisationAgreementViewModel>()
-                .ForMember(dest => dest.SignedDateText, opt => opt.Ignore())                             
+                .ForMember(dest => dest.SignedDateText, opt => opt.Ignore())
                 .ForMember(dest => dest.AccountLegalEntityPublicHashedId, opts => opts.Ignore());
 
-            CreateMap<AgreementTemplateDto, AgreementTemplateViewModel>()                
-                .ForMember(dest => dest.PublishedDate , opt => opt.MapFrom(src => GetPublishedDate(src)));                
+            CreateMap<AgreementTemplateDto, AgreementTemplateViewModel>()
+                .ForMember(dest => dest.PublishedDate, opt => opt.Ignore());
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs:51]: need to remove this line else it won't map the date across

File: src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs [hunk 4]
```diff
@@ -47,9 +41,6 @@ public AgreementMappings()
                 .ForMember(dest => dest.SignedDateText, opt => opt.Ignore())
                 .ForMember(dest => dest.AccountLegalEntityPublicHashedId, opts => opts.Ignore());
 
-            CreateMap<AgreementTemplateDto, AgreementTemplateViewModel>()
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs]: Still require this line:
`CreateMap<AgreementTemplateDto, AgreementTemplateViewModel>();`

**Comment by shomavg** [comment]
> The EmployerAgreementTemplate being read back from the database hasn't been extended to include the new PublishedDate property so won't be mapped to the DTO and viewModel

Added the PublishedDate. Can you rereview please.

**Pull request merged**